### PR TITLE
*: add mirror dependency on CPU

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

as tittle

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lijie0123 — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: hardware-and-software-requirements.md [hunk 1]
```diff
@@ -52,6 +52,13 @@ TiDB 作为一款开源一栈式实时 HTAP 数据库，可以很好地部署和
 
 运行时所需的依赖库：glibc（2.28-151.el8 版本）
 
+### 镜像依赖
+
+支持的 CPU 架构如下：
+
+- x86_64，从 TiDB v6.6.0 开始，需要 [x84-64-v2 指令集](https://developers.redhat.com/blog/2021/01/05/building-red-hat-enterprise-linux-9-for-the-x86-64-v2-microarchitecture-level#)
```
Comment (hfxsd) [inline hardware-and-software-requirements.md:59]:
```suggestion
- x86_64，从 TiDB v6.6.0 开始，需要 [x84-64-v2 指令集](https://developers.redhat.com/blog/2021/01/05/building-red-hat-enterprise-linux-9-for-the-x86-64-v2-microarchitecture-level)
```

File: hardware-and-software-requirements.md [hunk 2]
```diff
@@ -52,6 +52,13 @@ TiDB 作为一款开源一栈式实时 HTAP 数据库，可以很好地部署和
 
 运行时所需的依赖库：glibc（2.28-151.el8 版本）
 
+### 镜像依赖
```
Comment (hfxsd) [inline hardware-and-software-requirements.md:55]:
```suggestion
### Docker 镜像依赖
```

**Pull request merged**